### PR TITLE
Feature/10647 aks pipeline access

### PR DESCRIPTION
**Pull request by gavinwakeley1**
Task: [AB#10647](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10647)
Story: [AB#10643](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/10643)

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -458,3 +489,14 @@ jobs:
...
+  - task: AzureCLI@2
+    name: enableEnhancedAKSFirewall
+    displayName: Enable Enhanced AKS Firewall Restrictions
+    inputs:
+      azureSubscription: 'NHSAPP-BuyingCatalogue (Non-Prod)'
+      scriptType: 'bash'
+      scriptLocation: 'inlineScript'
+      inlineScript: |
+          ipRange=$(az keyvault secret show --vault-name gpitfutures-dev-core-kv --name "gpitdevbjssvpn" --query value -o tsv)
+          az aks update -n gpitfutures-development-aks -g gpitfutures-development-rg-aks --api-server-authorized-ip-ranges "$ipRange" > /dev/null
```
Comment (PhysicsJon) [inline azure-pipelines.yml:502]: eof newline

File: housekeeping/azure-pipelines.yml [hunk 2]
```diff
@@ -302,3 +321,23 @@ jobs:
...
+            az aks update -n gpitfutures-development-aks -g gpitfutures-development-rg-aks --api-server-authorized-ip-ranges "$ipRange" > /dev/null
+
+    - task: AzureCLI@2
+      name: enableEnhancedStorageFirewall
+      displayName: Enable Enhanced SA Firewall Restrictions
+      inputs:
+        azureSubscription: 'NHSAPP-BuyingCatalogue (Non-Prod)'
+        scriptType: 'bash'
+        scriptLocation: 'inlineScript'
+        inlineScript: az storage account update --resource-group gpitfutures-development-rg-sa --name gpitfuturesdevelopment --default-action Deny > /dev/null
```
Comment (PhysicsJon) [inline housekeeping/azure-pipelines.yml:343]: eof newline

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Just the end of file newlines

File: housekeeping/azure-pipelines.yml (hunk 2 above)
Comment (gavinwakeley1) [reply housekeeping/azure-pipelines.yml]: done

File: azure-pipelines.yml (hunk 1 above)
Comment (gavinwakeley1) [reply azure-pipelines.yml]: done

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**